tests/console/aplay.pm

```diff
@@ -14,10 +14,12 @@ use base "consoletest";
 use testapi;
 use strict;
 use warnings;
+use utils 'ensure_serialdev_permissions';
 
 sub run {
     my $self = shift;
     select_console 'root-console';
+    ensure_serialdev_permissions;
```

Comment:
We already call it in `tests/console/system_prepare`, why do we need to call it again?

Reply:
http://eris.suse.cz/tests/12563#step/aplay/23

This happened even after I have removed reset_consoles in kmp_modules.pm